packages/security-cordova/plugin.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version='1.0' encoding='utf-8'?>
+<plugin
+    id="cordova-plugin-aerogear-security"
+    version="0.1.0"
```

Comment:
I meant whenever the version is bumped in the plugin's package.json, since it won't be updated automatically by lerna. For this particular case, both are `0.1.0` so all good, sorry for late response!